Tests/AppTests/MarkdownHTMLConverterTests.swift

```diff
@@ -6,7 +6,7 @@ import XCTest
 import Plot
 
 
-class MarkdownHTMLConverterTests: XCTestCase {
+class MarkdownHTMLConverterTests: WebpageSnapshotTestCase {
```

Comment:
One thing to bear in mind is that `AppTestCase` (which `WebpageSnapshotTestCase` extends) recreates the schema and is noticeable slower than `XCTestCase` (~150ms per test on my machine). If we start adding lots of snapshot tests that don't need the db we should probably disentangle that.